resources/views/service/printables/families.blade.php

```diff
@@ -30,7 +30,7 @@
         @foreach ($chunk as $registration)
         <tr>
           <td>{{ $registration->family->carers->first()->name }}</td>
-          <td>ARC00123</td>
+          <td>{{ $registration->family->rvid }}</td>
```

Comment:
Is it called something else now or still rvid?

Reply:
$family->rvid is now a magical dynamically constructed attribute!